p2p/kademlia.py

```diff
@@ -559,7 +559,7 @@ def refresh_idle_buckets(self):
             asyncio.ensure_future(self.lookup(rid))
 
     def _mkpingid(self, token: AnyStr, node: Node) -> bytes:
-        pid = force_bytes(token) + node.pubkey.to_bytes()
+        pid = to_bytes(text=token) + node.pubkey.to_bytes()
```

Comment:
This seems potentially broken since `token` can be either `bytes` or `str` (see [`typing.AnyStr`](https://docs.python.org/3/library/typing.html#typing.AnyStr)).

Reply:
Ah thanks for pointing that out, I didn't look up the def'n and it surprised me.

I'll add a test to catch it, and fix it.